modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -186,6 +186,18 @@
 	.e-n-tabs-content {
 		border: 1px none var( --n-tabs-border-color );
 		border-width: var( --n-tabs-border-width );
+
+		.e-collapse {
+
+			&:not(:first-child) {
+				  margin-top: var( --n-tabs-title-gap );
+			}
+
+			&.e-active {
+				  margin-bottom: var( --n-tabs-gap );
+			}
+		}
+
```

Comment:
Remove empty line. 
This one is important. 
In SCSS we we never add empty closing lines (please compare with other scss files).

Reply:
Done